src/server/object_services/object_server.js

```diff
@@ -783,11 +783,21 @@ async function read_object_md(req) {
         throw new RpcError('UNAUTHORIZED', 'read_object_md: role should be admin');
     }
 
+    let req_query = null;
```

Comment:
on future conditions we might need more data on req object. I think its better to move this to a function, something like _convert_rpc_params_to_bucket_policy_req(rpc_params)

Reply:
done